### ISSUE TITLE
Bug when compiling

### ISSUE THREAD

**Issue by Atronix1902** [mrcrowl/vscode-easy-less#75]
Hey, just found a bug, which occurred since version 1.7.0.
The compiler throws an error when I try to use my for-loop.
![image](https://user-images.githubusercontent.com/49077834/106359385-d5a73b80-6312-11eb-989e-9729d7873aa8.png)
![image](https://user-images.githubusercontent.com/49077834/106359409-f374a080-6312-11eb-8884-953e3659623e.png)

Since this still worked on version 1.6.3, I don't think it is my fault.

**Comment by mrcrowl** [comment]
Certainly not your fault 😃... this is due to an [upstream change introduced in less v4.x](https://github.com/less/less.js/releases/tag/v4.0.0) where the `math` mode is `"parens-division"` by default (meaning that you have to wrap any math expression with a `/` in parentheses).

The default for less v3.x was `"always"`, leading to the difference in behavior that you're experiencing.

Two ways to fix this:
 1. Wrap the expression containing `/` in parentheses: i.e. `@i: (@index * 5 / 100);`
 2. Add a `math` setting to your `settings.json`: e.g.
    ```json
    {
      "less.compile": {
        "math": "always"
      }
    }
    ```

Ideally, you'd be able to set the `math` option in a per-file directive, but I've just tested and that isn't working atm. I'll try and get a separate release out for that.

**Comment by mrcrowl** [comment]
Per-file directive is now available in [Easy Less v1.7.2](https://github.com/mrcrowl/vscode-easy-less/releases/tag/v1.7.2).
